plugins/woocommerce/templates/myaccount/form-edit-account.php

```diff
@@ -12,35 +12,59 @@
  *
  * @see https://woocommerce.com/document/template-structure/
  * @package WooCommerce\Templates
- * @version 8.7.0
+ * @version 9.3.0
  */
 
 defined( 'ABSPATH' ) || exit;
 
-do_action( 'woocommerce_before_edit_account_form' ); ?>
+/**
+ * Hook - woocommerce_before_edit_account_form.
+ *
+ * @since 9.3.0
```

Comment:
This hook wasn't added in 9.3.0. In doing some digging, it was introduced in this PR: https://github.com/woocommerce/woocommerce/pull/10975 which seems to be part of the 2.6.0 release
```suggestion
 * @since 2.6.0
```